src/dsp_tools/models/custom_warnings.py

```diff
@@ -19,7 +32,16 @@ class DspToolsUserWarning(Warning):
     @classmethod
     def showwarning(cls, message: str) -> None:
         """Print the warning, without context"""
-        cprint(f"WARNING: {message}", color="red", attrs=["bold"])
+        print(BOLD_RED + f"WARNING: {message}" + RESET_TO_DEFAULT)
+
+
+class DspToolsUserInfo(Warning):
+    """Class for general user-facing warnings"""
+
+    @classmethod
+    def showwarning(cls, message: str) -> None:
+        """Print the warning, without context"""
+        print(YELLOW + f"INFO: {message}" + RESET_TO_DEFAULT)
```

Comment:
Here, the mix between + concatenation and f-string is especially unelegant.

Reply:
what I like about it is that it clearly and visibly separates what the user message is and what is configuration